src/tests/ftest/pool/pool_svc.py

```diff
@@ -93,8 +92,7 @@ def test_poolsvc(self):
                     self.pool.connect()
                     self.pool.disconnect()
                     # kill another server which is not a leader and exclude it
-                    server = DaosServer(self.context, self.server_group, 3)
-                    server.kill(1)
+                    self.pool.start_rebuild([3], self.test_log)
```

Comment:
The start_rebuild name is confusing; it should reflect what the function really does, like kill_servers or kill_ranks. But not a problem of this PR.